nginx/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# a minimal Nginx container including containerbuddy and a simple virtualhost config
```

Comment:
Let me [turn the question around](https://github.com/tgross/triton-cloudflare/pull/1#discussion_r43663537): how would you structure this repo if you were building it to use for yourself? Would you push the Dockerfile and config files into a subdirectory? Or would you put those up top?